### DISCUSSION TITLE
Maven project with modules: target directory not ignored when synchronize super POM project

### DISCUSSION THREAD

**Discussion by damianozucconi** [subclipse/subclipse#270] [Q&A]
Hi,

when I synchronize Maven "container" project with SVN (that contains many module sub-projects) the view doesn't filter `target` folders of sub projects:
![image](https://github.com/subclipse/subclipse/assets/12892164/99c6f702-f708-4a42-81a0-d40bbf75a1ea)

if I synchronize a single module subproject, the target folder is correctly ignored.
Is this an issue ? Is there a way to force ignore target folder without use ignore it on SVN ?

**Comment by markphip** [comment]
This project is only minimally maintained and the Synchronize view has always been a nightmare so this one would have little chance of ever being resolved.

Eclipse tells us what to ignore. The issue with Maven builds is they happen outside of Eclipse's view and so it does not get a chance to mark the compiled files as something to ignore. The best option is likely to add target folder to svn:ignore

**Reply by damianozucconi** [reply]
Understood 😕
Thanks for your reply
